### ISSUE TITLE
create MeasureTransport account at quay.io (or somewhere else) and add organization secrets

### ISSUE THREAD

**Issue by bonh** [MeasureTransport/MParT-examples#22]

**Comment by mparno** [comment]
@bonh What do you need me to do here?   Just create the account?

**Comment by bonh** [comment]
Basically, yes, but we need to decide first
1. wether we event want to provide Docker container images, 
2. which registry we want to use.

**Comment by dannys4** [comment]
I would advocate for the Docker images, particularly if they're being updated with each new release/merged PR. I have no prior knowledge or preference on 2. People seem to prefer Quay in terms of usability, and it's [explicitly used](https://fenics.readthedocs.io/projects/containers/en/latest/work_flows.html#separate-container-for-each-user-project) in the FEniCS docs. I say that, if it's good enough for them it's probably good enough for us.

**Comment by mparno** [comment]
I've only used dockerhub in the past, but am more than happy to use quay for MParT as long as we can set up a trigger to automatically build the images when a new release is created on github.

**Comment by bonh** [comment]
Yes, the image is automatically built from [this action](https://github.com/MeasureTransport/MParT/blob/e366960c2ae75a74eec7f539729ed99df248811b/.github/workflows/build-push-docker.yml), part of #https://github.com/MeasureTransport/MParT/pull/192/files. Right now this action is executed on push to branch and tags the resulting image on quay as `mpart_base:latest` but I'm sure we can add git tags to the image tag too.

**Comment by bonh** [comment]
I created the organization `measuretransport` as a test; we can change that. And I would like to move it from my personal account to something more general, perhaps `measuretransport@mit.edu`?

**Comment by dannys4** [comment]
I can make an MIT mailing list pretty easily if you'd like

**Comment by bonh** [comment]
Mailing list doesn't sound right. I think it should be a real MIT account collecting quay and github admin stuff. I don't know how something like this is normally handled.

**Comment by bonh** [comment]
Ah, right, I understand. To create an organization at quay I had to create a personal account and then create an organization with an additional email. I am using a personal address for the organization too and I want to replace that with a different address. If you think it will work with something like a mailing list I will change it.
